src/Http.ts

```diff
@@ -49,6 +37,11 @@ export default class Http {
     }
   }
 
+  errorMessage(err: string) {
+    this.error = { error: Error(err).message }
```

Comment:
@igorbezsmertnyi 
Do we need to store the last error message in `this.error`? `errorMessage()` may be useful later to adapt errors from Spree, but I don't think we'll need `this.error`.

Please let me know if you plan on using it somehow.